helm/gateway-api-config/values.yaml

```diff
@@ -46,8 +46,7 @@ gateways:
     dnsName: gateway
     className: giantswarm-default
     overrideBaseDomain: ""
-    subdomains: []
-    certificate:
```

Comment:
wdyt if we moved the certificate block into the listener. this way we could:
- disable the cert creation for one particular listener
- use different issuers on each listener